### ISSUE TITLE
Vehicle Spawns Where Player Stands + Depot Not Charging

### ISSUE THREAD

**Issue by DannysRP** [JonasDev99/qb-garages#65]
On the most recent update even with all the correct config settings set the vehicle will always spawn where you are standing and not in a parking spot. I'll even get an error saying I am not close enough to a parking spot but it pulls the vehicle out anyways. Also impound lot is not taking money upon pulling vehicle out of the impound lot. (Tried this with my modified version and fresh install)

**Comment by DannysRP** [comment]
After doing the recent download this is still happening. Vehicles spawn wherever you stand and not in the parking spots. Does this for all garages and impound lots. It also still doesn't charge you the impound lot fee when retrieving the vehicle.

**Comment by DannysRP** [comment]
Just tested this on a fresh txAdmin server and fresh resource still both errors persist

**Comment by TonybynMp4** [comment]
hmm das weird

**Comment by JonasDev99** [comment]
Should be working fine now, could you test the latest version?

**Comment by DannysRP** [comment]
I have tested the latest version and it still spawns the vehicle where the player stands. If you are close to a parking spot it will pull it out there but if you are not it will just pull it out where you are standing

**Comment by JonasDev99** [comment]
Is this still an issue?

**Comment by DannysRP** [comment]
I did the latest download and when I go to store / retrieve a vehicle it now does nothing. No errors from what I can see

**Comment by buddizer** [comment]
Working fine for me. It is true though that if you are standing on specific parking lot and do not have spawn in vehicle option enabled, the vehicle will spawn on top and you will have to move and then get in.

**Comment by DannysRP** [comment]
Yeah it doesn't do that at all. It just does nothing at all when trying to use the garage. I could send you my copy for a test
